components/arena/docker/arena_launcher.py

```diff
@@ -190,6 +191,12 @@ def generate_job_command(args):
       for e in env:
         commandArray.append("--env={0}".format(e))
 
+    if len(sync_source) > 0:
+      if not sync_source.endswith(".git"):
+        raise ValueError("sync_source must be an http git url")
+      commandArray.append('--sync-mode=git')
+      commandArray.append('--sync-source='+sync_source)
```

Comment:
Does it accept arguments without '='?
It might be more flexible this way.
```suggestion
      commandArray.extend(['--sync-source', sync_source])
```

Reply:
Yes, I've updated according to your suggestions! Thanks.